docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchSelectionTwo.js

```diff
@@ -3,6 +3,16 @@ import { Dropdown } from 'semantic-ui-react'
 
 import { stateOptions } from '../common'
 
+/** stateOptions shape **/
+/**
+[
+  { value: 'AL', text: 'Alabama' },
```

Comment:
Let's shorten all examples that have single-line option objects:

```js
/** stateOptions shape **/
/**
[
  { value: 'AL', text: 'Alabama' },
  { value: 'AK', text: 'Alaska' },
  { value: 'AZ', text: 'Arizona' },
  ...
]
**/
```
Into this style comment:
```js
// stateOptions = [{ value: 'AL', text: 'Alabama' }, ...{}]
```